content/population-ethics.md

```diff
@@ -420,9 +420,9 @@ The next chapter discusses the most important implications of utilitarianism for
 [^44]:
-    The critical range theories we have discussed are more difficult to capture in a numerical formula. But if we put aside any lives that fall within the critical range, we might approximate the remaining overall value by separately summing the positive and negative welfare levels, to the extent that they exceed their respective ends of the critical range. So, using positive and negative subscripts to denote the respective numbers, averages, and critical boundary points for these two subpopulations, we can write the value function as: <strong>N<sub>+</sub> _ (Q<sub>+</sub> - α<sub>+</sub>) + N<sub>-</sub> _ (Q<sub>-</sub> - α<sub>-</sub>)</strong>.
+    The critical range theories we have discussed are more difficult to capture in a numerical formula. But if we put aside any lives that fall within the critical range, we might approximate the remaining overall value by separately summing the positive and negative welfare levels, to the extent that they exceed their respective ends of the critical range. So, using positive and negative subscripts to denote the respective numbers (N), averages (Q), and critical boundary points (α) for these two subpopulations, we can write the value function as: <strong>N<sub>+</sub> * (Q<sub>+</sub> - α<sub>+</sub>) + N<sub>-</sub> * (Q<sub>-</sub> - α<sub>-</sub>)</strong>.
 
-    Alternatively, we might think of the value of the world as itself indeterminate or corresponding to a numerical range, as given by all the possible critical level theories corresponding to the critical range, i.e.: everything from <strong>N _ (Q - α<sub>-</sub>) </strong>to <strong>N _ (Q - α<sub>+</sub>)</strong>. On this approach, the value of the repugnant world Z, for example, would be indeterminate across an immense range of values stemming from <em>extremely negative</em> to <em>extremely positive</em>, because <strong>N</strong> is unimaginably huge whereas the sign of <strong>(Q - α)</strong> is indeterminate, given the range of candidate critical levels <strong>α</strong>.
+    Alternatively, we might think of the value of the world as itself indeterminate or corresponding to a numerical range, as given by all the possible critical level theories corresponding to the critical range, i.e.: everything from <strong>N * (Q - α<sub>-</sub>) </strong>to <strong>N * (Q - α<sub>+</sub>)</strong>. On this approach, the value of the repugnant world Z, for example, would be indeterminate across an immense range of values stemming from <em>extremely negative</em> to <em>extremely positive</em>, because <strong>N</strong> is unimaginably huge whereas the sign of <strong>(Q - α)</strong> is indeterminate, given the range of candidate critical levels <strong>α</strong>.
```

Comment:
Thank you for catching that -- an overly aggressive auto-formatter seems to have "thought" that `*` is equivalent to `-` (is it is when making a list).

I think the best thing to do is to prepend it with `\` so as to "escape" the character, making sure the interpreter understands it as a string and not a _markdown_ feature 🤔 

So `*` should become `\*`

But I can fix that up later 👌

Reply:
Ok, thanks!